### PR TITLE
Add a gradient background to workflow links

### LINKED ISSUE #1151: Add completeness bar to workflow selection buttons
The design for the workflow select button in the Hero widget includes a completeness bar which needs to be added:

Screenshot 2019-09-20 13 56 54

Related: #1077 

And https://zooniverse.slack.com/archives/C0Z7YNTSL/p1554151344033200

### PR DESCRIPTION
**Pull request by eatyourgreens**
Pass the workflow completeness to workflow buttons, and render a background gradient based on that value.

TODO:
- [x] visual focus/hover state.
- [x] indicate the workflow completeness to screenreaders etc.
- [x] dark theme styles.

Package:
app-project

Closes #1151.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Comment by beckyrother** [comment]
@eatyourgreens is there a place I could see this in action?

**Comment by eatyourgreens** [comment]
Pull down the code to someone’s laptop and `docker-compose up` in the `app-project` directory, I’m afraid.

**Comment by rogerhutchings** [comment]
recent-classifications

**Comment by beckyrother** [comment]
Thank you @rogerhutchings! I made a note in #1151 but copied it here:

If we need to remove the blue tooltip, let's add a secondary line to the button to indicate the percent complete and give a reason that the button is a progress bar. 
image
Note the white 1px border between the gradient and the box background.
Gradient: #F0B200 to #D46256

**Comment by eatyourgreens** [comment]
Do you have the names for those colours in the theme? No worries if not, I’ll look them up tomorrow.

**Comment by beckyrother** [comment]
No, I don't have access to the theme info. Sorry!

**Comment by eatyourgreens** [comment]
I went with Tomato (`status-critical`) and Gold (`neutral-4`) from the theme. Here's what it looks like after updating the CSS for the background colours. I picked Nest Quest Go at random because they have a selection of workflows.
image

**Comment by eatyourgreens** [comment]
Here's those same workflows with left-aligned labels.
Screenshot of 3 workflows for Nest Quest Go, showing left-aligned labels for the workflow name and completeness percentage.

**Comment by eatyourgreens** [comment]
I'll open a separate PR that adds a story for the Hero, with some dummy workflows, because `fresnel` might hold that up.

File: packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/WorkflowSelectButton.js [hunk 1]
```diff
@@ -18,10 +27,24 @@ function WorkflowSelectButton (props) {
 
   const as = addQueryParams(url, router)
   const href = '/projects/[owner]/[project]/classify'
+  const completeness = parseInt(workflow.completeness * 100)
```
Comment (rogerhutchings) [inline packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/WorkflowSelectButton.js]: this needs a radix of 10:

`const completeness = parseInt(workflow.completeness * 100, 10)`

File: packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/theme.js [hunk 2]
```diff
@@ -17,18 +17,32 @@ const theme = {
       }
     },
     extend: props => {
-      const { theme: { dark, global: { colors } } } = props
+      const { theme: { global: { colors } }, completeness } = props
+      const percentComplete = `${parseInt(completeness * 100)}%`
```
Comment (rogerhutchings) [inline packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/theme.js]: do you need to do this again? if so, it needs the radix parameter again

Comment (eatyourgreens) [reply packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/theme.js]: Good point. Fixed now.

**Comment by rogerhutchings** [comment]
recent-classifications

**Comment by eatyourgreens** [comment]
From my earlier comment about the completed percentage, I've already reduced that text size.
>the text looks a bit big to me here, so I've knocked the font size down to xsmall.

I'm not sure if there's a size below `xsmall` though.

**Comment by srallen** [comment]
I'm curious about what this looks like with a very long workflow display name? Perhaps we can check that out in a story after this merges.

**Comment by eatyourgreens** [comment]
That's a good point. I've been picking projects at random to test this. The Hero component needs stories, but uses `fresnel`, which breaks the storybook.

**Comment by rogerhutchings** [comment]
Screenshot 2019-10-03 15 58 47

**Comment by beckyrother** [comment]
Thank you for the screenshot Roger, I can't see anything if it's not posted here. That text looks better.

**Comment by eatyourgreens** [comment]
Stories have been added, along with a mock router implementation to get the links working in the storybook.

**Pull request merged**